assets/l10n/ru-RU.ftl

```diff
@@ -426,6 +426,7 @@ fcm_user_removed_you =
        *[other] {$userName}
     } исключил Вас из группы
 fcm_webpush_link = /chats/{$chatId}
+hyphen = -
```

Comment:
Do you mind if I rename this from `dash` into `hyphen`? While the symbol itself has no differences, it's semantic meaning seems to be different for dash and hyphen:


![hyphens_vs_dashes](https://github.com/user-attachments/assets/186aabcc-76f0-49c1-80b1-cf639db45821)


And since we use that symbol as a part between quartets, I guess it's more of hyphen than a dash?